modules/test/playwright/pages/portal-workflow-kaleo-designer-web/ConfigurationTabPage.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, Page} from '@playwright/test';
+
+import {waitForSuccessAlert} from '../../utils/waitForSuccessAlert';
+
+export class ConfigurationTabPage {
+	readonly configurationTabLink: Locator;
+	readonly page: Page;
+	readonly workflowAssigned: Locator;
+	cancelButton: Locator;
+	editButton: Locator;
+	saveButton: Locator;
+
+	constructor(page: Page) {
+		this.page = page;
+		this.configurationTabLink = page.getByRole('link', {
+			name: 'Configuration',
+		});
+		this.workflowAssigned = page
+			.getByRole('cell', {name: 'Workflow Definition'})
+			.getByTitle('Workflow Definition');
+	}
+
+	async goTo() {
+		await this.configurationTabLink.waitFor({state: 'visible'});
+		await this.configurationTabLink.click({force: true});
+		await this.page.waitForURL((url) =>
+			url.href.includes('=configuration')
+		);
+		await this.page.reload();
```

Comment:
Hey Carlos, could you tell me if in all cases where we want to click, we will need to reload? Or would this be some specific check for your case?

Reply:
this configuration page is problematic. After much debugging, I found that to reliably interact with its elements you need to reload, so I am already doing it every time you land on this page.

Reply:
Hey Carlos, I saw that the test has not yet passed at the CI.

Also, as mentioned by you, one of the reloads was removed. However, we still have one more, could you check if, like the first one that was removed, this reload can be avoided? 

I'm thinking of some case where there is a bug in the application, where the workaround is to reload the page. 

Tests using this goTo would have difficulty identifying this type of bug. Since we would always be reloading the page in tests, which is not common behavior in the application.

Reply:
Hey Carlos, I'm wondering if like the first reload, this one can also be avoided.

I'm thinking of some case where there is a bug in the application, where the workaround is to reload the page. 

Tests using this goTo would have difficulty identifying this type of bug. Since we would always be reloading the page in tests.